### ISSUE TITLE
Blobs win the game way too fast

### ISSUE THREAD

**Issue by Killette2** [d3athrow/vgstation13#16617]
![screenshot_88](https://user-images.githubusercontent.com/5169310/33226363-14300484-d15a-11e7-9877-311347b89945.png)
A blob last round needed to be only about this big for the round to end. When it was over everyone in OOC were saying something along the lines of "is that it?". Nobody likes it when blob rounds are too long but this one felt like it was cut way too short.

**Comment by Killette2** [comment]
![image](https://user-images.githubusercontent.com/5169310/33226411-b7d50ed6-d15a-11e7-9ec1-e6ebda85aa0b.png)
OOC to prove I'm not talking out of my ass

**Comment by Pathid** [comment]
Yeah, it kills a lot of the drama when the blab can "take over" without affecting you or your department at all. And I imagine that on Deff you can probably have a blob win without absorbing any working parts of the station.

**Comment by AutismSpider** [comment]
blob just has to take over any part of the station not any of the main  departments is just lame on bigger maps.

**Comment by gurfan** [comment]
This this issue was made blob has gone through a lot of changes, and the size requirement is larger.
